static/js/src/public/details/integrations/components/InterfaceItem.tsx

```diff
@@ -161,15 +161,19 @@ export const InterfaceItem = ({
             </p>
           </div>
           <Row>
-            {charms.map((charm: ICharm) => (
-              <Col
-              size={3}
-              style={{ marginBottom: "1.5rem" }}
-              key={charm.package["display_name"]}
-              >
-                <IntegrationCard data={charm} />
-              </Col>
-            ))}
+            {charms
```

Comment:
Spicy - how are we getting duplicates that then need filtering out? Does the API return duplicates? If the API doesn't we should probably fix the root cause in our code, rather than masking it with a filter.
If it is because we get duplicates from the API, it would be useful to have a comment here explaining why we have this and letting the store team know there are duplicates.

Reply:
I've been digging to this, and I think `<IntegrationCard>` might be doing some of its own state management or manipulation under the hood. When I log out `charms` right before this, there is only one of each kind, but when rendering using `<IntegrationCard>`, there are duplicates :thinking: